publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/CliOptions.java

```diff
@@ -84,7 +84,7 @@ public static void createCLIOptions() {
         options.addOption("v", "lists the versions of publish and all loaded protocols");
         options.addOption("tag", "tag", true, "tag to be used in routing key");
         options.addOption("rk", "routing_key", true, "routing key of the eiffel message. When provided routing key is not generated and the value provided is used.");
-
+        options.addOption("tto", "tcp_time_out", true, "specifies tcp connection timeout, default time is 60 seconds");
```

Comment:
I believe the unit for "tto" option should be given in milliseconds, not seconds, right? It isn't quite clear here.

In RabbitMqProperties.java:
//  built in tcp connection timeout value for MB in milliseconds.
    public static final Integer DEFAULT_TCP_TIMEOUT=60000;